webrender/src/prim_store.rs

```diff
@@ -1653,6 +1653,10 @@ impl PrimitiveStore {
                     Some(pic_rect)
                 };
 
+                if !pic_state_for_children.is_cacheable {
+                  pic_state.is_cacheable = false;
```

Comment:
why aren't children cachable if the parent is not? If a picture contains an external image and another picture, that other picture might as well be cached, no?

Reply:
The intention is the reverse: If a picture contains a picture, and that inner picture contains an external image, then neither picture is cacheable.